src/backend/src/routes/work-packages.routes.ts

```diff
@@ -66,5 +66,6 @@ workPackagesRouter.post(
 );
 
 workPackagesRouter.get('/template/:workPackageTemplateId', WorkPackagesController.getSingleWorkPackageTemplate);
+workPackagesRouter.get('/template', WorkPackagesController.getAllWorkPackageTemplates);
```

Comment:
there's overlap between the routes rn, because `:wbsNum` is a URL param, it'll overlap with `template`, so we'll need to make this route: '/all-templates/get'